### PR TITLE
fix: Issue #1374 All suggestions are being displayed with scroll area

### LINKED ISSUE #1374: Bug: Not all suggestions are being displayed on insight page
### Describe the bug

Just few repos are being showed when a user searches for a repo even tho the query returns many repos.

Screenshot (782)

### Steps to reproduce

-  Go to the page to create a new insight, [https://insights.opensauced.pizza/hub/insights/new](https://insights.opensauced.pizza/hub/insights/new)
- Open dev tools and click on the Network tab

### Browsers

_No response_

### Additional context (Is this in dev or production?)

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by aryanas159**
## Description

fixes #1374 
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style

https://github.com/open-sauced/insights/assets/114330931/481250ba-13d7-4b60-bbb1-0cc3594bc7a4

- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by babblebey** [comment]
@aryanas159 

Please add a Screencast (video) to this PR that demos your implementation to aid review.

**Comment by aryanas159** [comment]
@babblebey I have added the screencast.

**Review by OgDev-01 — APPROVED** [review]
Thanks for the fix @aryanas159 🍕 LGTM

File: components/atoms/Search/search.tsx [hunk 1]
```diff
@@ -83,19 +85,22 @@ const Search = ({
...
-              className="px-4 py-2 overflow-hidden break-all text-light-slate-9 hover:bg-light-slate-2"
-              style={suggestionsStyle}
-              key={index}
-              onClick={() => handleOnSelect(suggestion)}
-            >
-              <span className="pl-5 text-sm ">{suggestion}</span>
-            </div>
-          ))}
+          <ScrollArea type="auto">
+            <ScrollViewport className="max-h-60 w-full">
```
Comment (babblebey) [inline components/atoms/Search/search.tsx:91]: Remove the usage of `ScrollViewport`, use only the `ScrollArea` with a fixed height

```javascript
<ScrollArea type="auto" className="h-60">
  ...
</ScrollArea>
```
Using the `ScrollViewport` assumes you want extra customization to the T! but you don't have to.

**Review by babblebey — CHANGES_REQUESTED** [review]
There's another instance of suggestion counts that you have to fix:

https://github.com/open-sauced/insights/blob/beta/components/molecules/TableHeader/table-header.tsx#L67

Remove the line

```javascript
if (suggestions.length > 5) suggestions.length = 5;
```

**Comment by aryanas159** [comment]
@babblebey  Thanks for the suggestion, I was confused why I couldn't see the scrollbar. I have made the required changes.

https://github.com/open-sauced/insights/assets/114330931/2896ae0c-adc8-4ba5-9b20-87890cdd33e6

**Review by bdougie — APPROVED** [review]
This is a small improvement, but I would like to not the overall suggestions experience needs better data fetching and a deeper look at the UX. I expect that to happen here on the PR, but would love to discuss a better UI in an issue.

**Pull request merged**